cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -0,0 +1,229 @@
+
+DELPHI data access rules are available [here](/record/417), or from [the DELPHI web pages](https://delphi-www.web.cern.ch/delphi-www/delsec/finalrules/DELPHI_Data_preservation-8.pdf).
+
+Please read these before accessing the DELPHI software and data.
+
+## Accessing the software stack
+There are two possible ways to access the software stack.
+
+### Docker
+A docker container is available which ships with all the modules installed. Please take a look [here](docs/delpi-guide-docker)
+
+### CVMFS
+Binaries are also available from CVMFS, for a variety of different Linux flavors. There are no native Windows or Apple ports available for the time being.
+
+This method is convenient if you are running a desktop or a virtual machine with one of the supported Linux flavors on it.
+
+#### Requirements
+You will need to have:
+
+* /cvmfs/delphi.cern.ch mounted
+* /eos/opendata/delphi (or /eos/experiments/delphi) mounted
+* A list of additional packages installed, see https://gitlab.cern.ch/delphi/deployment for more information:
+    * general: tcsh xfonts-100dpi xfonts-75dpi libxfont2
+    * compilers: cmake gcc g++ gfortran
+    * library packages: libx11 libglu1-mesa xutils libmotif r-base xutils libxbae libxaw7 libssl libglew libdlm
+
+Please check the instructions at https://cvmfs.readthedocs.io/en/stable/cpt-quickstart.html and https://eoscta.docs.cern.ch/install/eos/ and https://eos-web.web.cern.ch/eos-web.
+
+#### Initialising the environment:
+For C-Shell (csh, tcsh ), do
+```
+ source /cvmfs/delphi.cern.ch/setup.csh
+```
+
+For Bourne shell (sh, bash, zsh, ...), type
+```
+. /cvmfs/delphi.cern.ch/setup.sh
+```
+If using the container, please use ```/etc/profile.d/delphi.sh``` or ```/etc/profile.d/delphi.csh``` instead.
+
+After sourcing the environment, make sure that the environment variable **DELPHI_DATA_ROOT** points to the tip of the data area. This is specifically important if you copied the data sets you want to use locally. The default is ```/eos/experiment/delphi/castor2015```.
+
+## Documentation
+DELPHI manuals and notes are available from [https://cds.cern.ch/](http://cds.cern.ch/search?ln=en&p=&action_search=Search&op1=a&m1=a&p1=&f1=&c=DELPHI&sf=&so=d&rm=&rg=10&sc=1&of=hb)
+
+Here is a selection for getting started:
+
+* [Analysis framework](/docs/27002)
+* [Event display](/docs/27003)
+* [DST contents](/docs/27004)
+
+## Source code
+The sources are available on https://gitlab.cern.ch/delphi. Some modules still requires CERN authentication. The plan is to release the software in the near future.
+
+## Compilers:
+The DELPHI stack is mostly written in Fortran, with some bits written in C. Only gfortran and GNU gcc are supported. We use the gfortran version which comes with the supported operating system.
+
+## Getting help
+
+The collaboration main contact for data preservation is the mailing list DELPHI-data-preservation-board@cern.ch. Support can only be given on a best effort basis. Suggestions and feedback is of course welcome!
```

Comment:
Suggestion: Shall we direct people to the Open Data Forum?  https://opendata-forum.cern.ch/ We try to send people there for "mutual help" and also to see past ansnwered questions. This may be more practical than an email addresss? We have ATLAS, CMS, LHCb communities at the Forum, and could create a DELPHI one if you are interested.

Reply:
I think this is something to be considered later. Getting in touch with the collaboration via this e-mail is for us a good way to redirect people and try to help them. For LHC experiments this is for sure different than for LEP experiments (or JADE which is even older).

Let's reconsider this at a later stage.